### PR TITLE
 doctest update metrics 

### PR DESCRIPTION
**Pull request by nataliyah123**
Fixes part of # 2066

### CODE REVIEW

File: tensorflow_addons/metrics/cohens_kappa.py [hunk 1]
```diff
@@ -38,28 +38,30 @@ class CohenKappa(Metric):
...
 
-    # To use this with weights, sample_weight argument can be used.
-    m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
-    m.update_state(actuals, preds, sample_weight=weights)
-    print('Final result: ', m.result().numpy()) # Result: 0.37209308
-    ```
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.61904764
```
Comment (WindQAQ) [inline tensorflow_addons/metrics/cohens_kappa.py:49]: Why these are commented out?

File: tensorflow_addons/metrics/cohens_kappa.py [hunk 2]
```diff
@@ -38,28 +38,30 @@ class CohenKappa(Metric):
...
+
+    >>> # To use this with weights, sample_weight argument can be used.
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds, sample_weight=weights)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.37209308
```
Comment (WindQAQ) [inline tensorflow_addons/metrics/cohens_kappa.py:55]: ditto

File: tensorflow_addons/metrics/matthews_correlation_coefficient.py [hunk 3]
```diff
@@ -42,18 +42,18 @@ class MatthewsCorrelationCoefficient(tf.keras.metrics.Metric):
...
-    mcc.result().numpy())
-    # Matthews correlation coefficient is : -0.33333334
-    ```
+
+    >>> actuals = tf.constant([[1.0], [1.0], [1.0], [0.0]],
+    ... dtype=tf.float32)
+    >>> preds = tf.constant([[1.0], [0.0], [1.0], [1.0]],
+    ... dtype=tf.float32)
+    >>> # Matthews correlation coefficient
+    >>> mcc = MatthewsCorrelationCoefficient(num_classes=1)
```
Comment (WindQAQ) [inline tensorflow_addons/metrics/matthews_correlation_coefficient.py:51]: `tfa.metrics.MatthewsCorrelationCoefficient`?

**Review by WindQAQ — CHANGES_REQUESTED** [review]
Thanks for the contribution! Can we canonicalize the variable names? Like

- `actuals` -> `y_true`
- `preds` -> `y_pred`
- all metric instance should be named `metric`
- all results from `metric(y_true, y_pred)` should be named `result`.

File: tensorflow_addons/metrics/cohens_kappa.py (hunk 1 above)
Comment (nataliyah123) [reply tensorflow_addons/metrics/cohens_kappa.py:49]: please see the image below. 
I don't reckon what do you mean by 'all results from metric(y_true, y_pred) should be named result'. do you mean whenever a metric is directly called or do you mean metric.update_state(y,ypred), should be assigned to result?. if I assign update_state to result the print statement should be like
`print('Final result: ', result.result().numpy())` 
can you please check this [PR](https://github.com/tensorflow/addons/pull/2109). it has a similar issue as described below. I have not reverted back the changes 
image

File: tensorflow_addons/metrics/matthews_correlation_coefficient.py (hunk 3 above)
Comment (nataliyah123) [reply tensorflow_addons/metrics/matthews_correlation_coefficient.py]: I had my doubt about this but since it was working completely fine. So, I left it as it is.

File: tensorflow_addons/metrics/cohens_kappa.py (hunk 2 above)
Comment (nataliyah123) [reply tensorflow_addons/metrics/cohens_kappa.py:55]: it's not complete ditto the second example uses weights.

**Comment by nataliyah123** [comment]
@WindQAQ can I change the pytest .ini file and remove DONT_ACCEPT_BLANKLINE because the output is not clearing the test

**Comment by Harsh188** [comment]
> @WindQAQ can I change the pytest .ini file and remove DONT_ACCEPT_BLANKLINE because the output is not clearing the test

Can you try adding `#doctest: -DONT_ACCEPT_BLANKLINE `

**Comment by nataliyah123** [comment]
working

**Comment by nataliyah123** [comment]
@googlebot I fixed it.

**Comment by nataliyah123** [comment]
@googlebot I fixed it.

**Comment by nataliyah123** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Faddons%2Fpull%2F2144) for more info**.
@googlebot I fixed it.

**Comment by nataliyah123** [comment]
> > We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> > In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> > **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Faddons%2Fpull%2F2144) for more info**.
> > @googlebot I fixed it.

@googlebot rescan

**Comment by nataliyah123** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Faddons%2Fpull%2F2144) for more info**.

@googlebot I fixed it.

**Comment by nataliyah123** [comment]
@googlebot I fixed it.

**Comment by WindQAQ** [comment]
This is probably stale as #2171 exists. Let me close it but feel free to reopen it if I make the mistake.
